packages/components/src/select-control/tags.js

```diff
@@ -21,6 +21,18 @@ class Tags extends Component {
 		super( props );
 		this.removeAll = this.removeAll.bind( this );
 		this.removeResult = this.removeResult.bind( this );
+		this.state = {
+			overflowActive: false
+		};
+	}
+
+	isElementOverflown(e) {
```

Comment:
This piece of code is removed since it no longer needed/used. CSS class that was used to add overflown properties was added to the original div element.